CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2824, Fix regression by reverting fix that returned 206 when first position = length in a `Range` header - @laurenceisla, @strengthless
  - #3015, Fix unnecessary count() on RPC returning single - @steve-chavez
 
+### Fixed
+
+ - #2824, Fix range request with 0 rows and 0 offset return status 416 - @strengthless
+ - #1070, Fix PUT responding with 200 when rows are inserted - @taimoorzaeem
```

Comment:
```suggestion
 - #1070, Fix PUT responding with 200 when rows are inserted - @taimoorzaeem
```

This must be under a single "Fixed" subtitle.  Rebase the latest change in main first and then check that only the current fix is added.

Reply:
Also, it needs to mention that `resolution=merge-duplicates` now returns `200` when no insert was done.